recipe/meta.yaml

```diff
@@ -22,7 +21,7 @@ requirements:
     - wheel
     - setuptools
   run:
-    - python >=3.6
+    - python   
```

Comment:
I think there's some trailing whitespace on this line after `python`. Not sure if it'll cause problems, but may be best to get rid of it.